eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -212,6 +212,17 @@ jobs:
 
       # ===== BEGIN LIVE TEST RUNS =====
 
+      - task: AzurePowerShell@5
+        inputs:
+          azureSubscription: ${{ parameters.CloudConfig.ServiceConnection }}
```

Comment:
Does this config need to be ran under the service connection? Currently that is only set if UseFederatedAuth is also set so we might need to include this step under that condition.

Reply:
Yes I copied from below but wasn't sure which inputs are needed.

Reply:
removed

Reply:
>  so we might need to include this step under that condition.

The step was under that condition but it doesn't applies to others like dependency testing. I will try whether running this command under normal powershell would also work.

Reply:
Reverting to original.  Seems that AzurePowerShell@5 requires that input. Normal pwsh doesn't have Az module loaded.

Reply:
If it is just the az modules and doesn't require login you can use https://github.com/Azure/azure-sdk-for-js/blob/main/eng/common/scripts/Import-AzModules.ps1 to import the az modules before calling.

Reply:
Thanks @weshaggard! This now works as expected within a pwsh